templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -190,5 +190,130 @@
 				</div>
 			{{end}}
 		{{end}}
+
+		{{if .IsSigned}}
+			{{if .IssueDependenciesEnabled}}
+		<div class="ui divider"></div>
+
+		<div class="ui depending">
+			<span class="text"><strong>{{.i18n.Tr "repo.issues.dependency.title"}}</strong></span>
+			<br>
+			{{if .BlockedByDependencies}}
+			<span class="text" data-tooltip="{{if .Issue.IsPull}}
+					{{.i18n.Tr "repo.issues.dependency.issue_closing_blockedby"}}
+				{{else}}
+					{{.i18n.Tr "repo.issues.dependency.pr_closing_blockedby"}}
+				{{end}}" data-inverted="">
+			{{.i18n.Tr "repo.issues.dependency.blocked_by_short"}}:
+			</span>
+			<div class="ui relaxed divided list">
+				{{range .BlockedByDependencies}}
+				<div class="item">
+					<div class="right floated content">
+						<a class="delete-dependency-button" onclick="deleteDependencyModal({{.Issue.ID}}, 'blockedBy');">
+							<i class="delete icon text red"></i>
+						</a>
+						{{if .IsClosed}}
+						<div class="ui red mini label">
+							<i class="octicon octicon-issue-closed"></i>
+						</div>
+						{{else}}
+						<div class="ui green mini label">
+							<i class="octicon octicon-issue-opened"></i>
+						</div>
+						{{end}}
+					</div>
+					<div class="ui black label">#{{.Index}}</div>
+					<a class="title has-emoji" href="{{$.RepoLink}}/issues/{{.Index}}">{{.Title}}</a>
+				</div>
+				{{end}}
+			</div>
+			{{end}}
+
+			{{if .BlockingDependencies}}
+			<span class="text" data-tooltip="{{if .Issue.IsPull}}
+					{{.i18n.Tr "repo.issues.dependency.pr_close_blocks"}}
+				{{else}}
+					{{.i18n.Tr "repo.issues.dependency.issue_close_blocks"}}
+				{{end}}" data-inverted="">
+			{{.i18n.Tr "repo.issues.dependency.blocks_short"}}:
+			</span>
+			<div class="ui relaxed divided list">
+				{{range .BlockingDependencies}}
+				<div class="item">
+					<div class="right floated content">
+						<a class="delete-dependency-button" onclick="deleteDependencyModal({{.Issue.ID}}, 'blocking');">
+							<i class="delete icon text red"></i>
+						</a>
+						{{if .IsClosed}}
+						<div class="ui red tiny label">
+							<i class="octicon octicon-issue-closed"></i>
+						</div>
+						{{else}}
+						<div class="ui green mini label">
+							<i class="octicon octicon-issue-opened"></i>
+						</div>
+						{{end}}
+					</div>
+					<div class="ui black label">#{{.Index}}</div>
+					<a class="title has-emoji" href="{{$.RepoLink}}/issues/{{.Index}}">{{.Title}}</a>
+				</div>
+				{{end}}
+			</div>
+			{{end}}
```

Comment:
Please fix the indention of this template. (I know that this is hard due to the lack of a good working beautifier)

Reply:
Done.